### DISCUSSION TITLE
Is it possible to connect two wallets (Ethereum and Solana) at the same time?

### DISCUSSION THREAD

**Discussion by in19farkt** [WalletConnect/walletconnect-monorepo#4551] [Developer Support] [answered]
For example, I want to create a bridge app and I need to connect an Ethereum wallet and a Solana wallet.

I wanted to use @web3modal/wagmi and @web3modal/solana, but I can't because both packages use the same variable `modal` from the @web3modal/scaffold-react package.

**Comment by glitch-txs** [comment, marked as answer]
Not supported at the moment, but we're working on it 👍

**Reply by app-matt** [reply]
Great to hear - is there any timeline you can share, or is it a way off?
